src/components/CollapsedOpeningTimes.js

```diff
@@ -74,7 +87,10 @@ export default class CollapsedOpeningTimes extends React.Component {
         ];
 
         let openingHours = _(this.props.object.openingTimes)
-            .sortBy(record => order.indexOf(record.day));
+            .sortBy(record =>
+                (order.indexOf(record.day) * 24) +
+                (parseInt(record.open))
```

Comment:
Huh strange. I tried it in the console and it was always 1 but now that I try more it's different. OK, ignore me.